doc/support.rst

```diff
@@ -62,14 +62,14 @@ Don't forget to include:
 
   - expected outcome,
 
-  - observed outcome or python (or gdb) tracebacks
+  - observed outcome or Python (or gdb) tracebacks
 
 To help developers fix your bug faster, please link to a https://gist.github.com
 holding a standalone minimalistic python script that reproduces your bug and
-optionally a minimalistic subsample of your dataset (for instance exported
+optionally a minimalistic subsample of your dataset (for instance, exported
 as CSV files using ``numpy.savetxt``).
 
-Note: gists are git cloneable repositories and thus you can use git to
+Note: Gists are Git cloneable repositories and thus you can use Git to
```

Comment:
I am not use to the capital G in git, but this is how [git's documentation](https://git-scm.com/about) does it.

I am okay with this change.

Reply:
Also not used top that but seems fine.